Content.Server/AI/Utility/Considerations/Containers/TargetAccessibleCon.cs

```diff
@@ -43,12 +43,7 @@ protected override float GetScore(Blackboard context)
 
             var owner = context.GetState<SelfState>().GetValue();
 
-            if (owner == null)
```

Comment:
So I don't know much about AI code. But the `SelfState` Owner value was never null even before the `IEntity` purge? But I guess `GetState` just always returned a null-able IEntity?

I've just removed several `owner == null` checks. If the owner is somehow being set to default somewhere that might still cause issues, but AFAICT that is not happening. And either way, these null checks wouldn't help with that.

Reply:
I'm gonna refactor how the blackboard works at some stage so I wouldn't worry about it.